_posts/en/newsletters/2025-04-25-newsletter.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Bitcoin Optech Newsletter #351'
+permalink: /en/newsletters/2025/04/25/
+name: 2025-04-25-newsletter
+slug: 2025-04-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Interactive aggregate signatures compatible with secp256k1:** Jonas
+  Nick, Tim Ruffing, Yannick Seurin [posted][nrs dahlias] to the
+  Bitcoin-Dev mailing list to announce a [paper][dahlias paper] they've
+  written about creating 64-byte aggregate signatures compatible with
+  the cryptographic primatives already used by Bitcoin.  Aggregate
+  signatures are the cryptographic requirement for [cross-input
+  signature aggregation][topic cisa] (CISA), a feature proposed for
+  Bitcoin that could reduce the size of transactions with multiple
+  inputs, which would reduce the cost many different types of
+  spending---including privacy-enhanced spending through
+  [coinjoins][topic coinjoin] and [payjoins][topic payjoin].
+
+  In addition to an aggregate signature scheme like the DahLIAS proposed
+  by the authors, adding support for CISA to Bitcoin would require a
+  consensus change and possible interactions between signature
```

Comment:
```suggestion
  consensus change, and possible interactions between signature
```

Reply:
Leaving as-is.  Although this could be an independent clause, I think it's also ok as one of the objects of the simple conjunction.